JetStream build errors on Windows

- [x] Defect
 - [ ] Feature Request or Change Proposal
 
## Defects 

jetstream branch build errors on Windows:
 ```
mem_windows.go:36:31: undefined: kernel32 
mem_windows.go:45:12: cannot use msx.ullTotalPhys (type uint64) as type int64 in return argument
```
 
#### Versions of `nats-server` and affected client libraries used:

jetstream branch

#### OS/Container environment:

Windows 10

#### Steps or code to reproduce the issue:

#### Expected result:
 
#### Actual result:

--- a/server/sysmem/mem_windows.go
+++ b/server/sysmem/mem_windows.go
@@ -29,7 +29,7 @@ type _memoryStatusEx struct {
 }
 
 func Memory() int64 {
-	k32, err := syscall.LoadDLL("kernel32.dll")
+	kernel32, err := syscall.LoadDLL("kernel32.dll")
 	if err != nil {
 		return 0
 	}
@@ -42,5 +42,5 @@ func Memory() int64 {
 	if res == 0 {
 		return 0
 	}
-	return msx.ullTotalPhys
+	return int64(msx.ullTotalPhys)
 }

